File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md

```diff
@@ -0,0 +1,127 @@
+
+## Customization of Border
+
+The following code snippet shows how to modify the border of the plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Format the Plot Area.
+IOfficeChartFrameFormat chartPlotArea = chart.PlotArea;
+//Plots area border settings - line pattern, color, weight
+chartPlotArea.Border.LinePattern = OfficeChartLinePattern.Solid;
+chartPlotArea.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chartPlotArea.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Color
+
+The following code snippet shows how to fill the color in plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+ //Sets fill type and color
+ chartPlotArea.Fill.FillType = OfficeFillType.Gradient;
+ chartPlotArea.Fill.GradientColorType = OfficeGradientColor.TwoColor;
+ chartPlotArea.Fill.BackColor = Syncfusion.Drawing.Color.FromArgb(205, 217, 234);
+ chartPlotArea.Fill.ForeColor = Syncfusion.Drawing.Color.White;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add Image in Plot Area
+
+The following code snippet shows how to fill the image in plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+FileStream imageStream = new FileStream("Data/Image.png", FileMode.Open, FileAccess.Read);
+Image image = Image.FromStream(imageStream);
+chartPlotArea.Fill.UserPicture(image, "image");
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Transparency level
+
+The following code snippet shows how to make transparency in plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+// Set the transparency of the plot area.
+chartPlotArea.Fill.Transparency = 0.5;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> [Transparency](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_Transparency) is only applicable when [FillType](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_FillType) is set as SolidColor. Color-shaded fill is represented as a floating-point value ranging from 0.0 (Clear) to 1.0 (Opaque).
+
+## Set the position of Plot Area
+
+The following code snippet shows how to make transparency in plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Sets position for plot area
+chartPlotArea.Layout.LeftMode = LayoutModes.auto;
+chartPlotArea.Layout.TopMode = LayoutModes.factor;
+chartPlotArea.Layout.LayoutTarget = LayoutTargets.outer;
+
+{% endhighlight %}
+{% endtabs %}
+
+The complete code snippet illustrating the above options is shown below.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+FileStream fileStreamPath = new FileStream("Data/Template.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open an existing document from file system through constructor of WordDocument class
```

Comment:
//Open an existing document from file system through constructor of WordDocument class.